src/main/java/eu/delta_source/internship/Cart.java

```diff
@@ -0,0 +1,91 @@
+package eu.delta_source.internship;
+
+import java.util.ArrayList;
+
+/**
+ * The Cart class is used to add delivery fee and VAT to the items from the Cart.
+ *
+ * @author Evgeni Georgiev
+ * @version 1.0
+ */
+public class Cart {
+	private final ArrayList<CartItem> cartItems = new ArrayList<>();
+	private double deliveryFee;
+
+	public double getDeliveryFee() {
+		return this.deliveryFee;
+	}
+
+	// Method outputting details for single product(label, price, quantity, result sum for price and quantity).
+	public void runProductsInCartOutput() {
+		cartItems.forEach(e -> System.out.println(e.runCartItemOutput()));
+	}
+
+	// Add new product to the cart.
+	public void addCartItem(CartItem cartItem) {
+		cartItems.add(cartItem);
+	}
+
+	// Method decreasing the count of quantity of single product by n-count(additional functionality).
+	// When the quantity is less than 1 the product is removed from the cart.
+	public void decreaseQuantityForItem(CartItem cartItem, int decreaseCountBy) {
+		int getQuantity = cartItem.getQuantity();
```

Comment:
'getQuanity' is good name for a method not for a variable. 
Good that you extracted it in a variable to reuse the variable instead of calling the method several times.